components/slider.css

```diff
@@ -0,0 +1,20 @@
+.slider-label{
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+}
+
+.slidecontainer {
```

Comment:
Just a suggestion slider- container could be a better name rather than slidecontainer